src/components/EditListingAvailabilityPanel/EditListingAvailabilityPanel.js

```diff
@@ -8,6 +8,9 @@ import { ListingLink } from '../../components';
 import { EditListingAvailabilityForm } from '../../forms';
 
 import css from './EditListingAvailabilityPanel.module.css';
+const LISTING_TYPE = {
+  equipment: 'equipment',
```

Comment:
This Listing_Type const was Duplicated in many files, why don't we declare it in only 1 constant file and export to other files?